tests/Integration/goldens/iam/src/V1/Client/BaseClient/IAMPolicyBaseClient.php

```diff
@@ -68,6 +68,8 @@
  * This class provides the ability to make remote calls to the backing service through method
  * calls that map to API methods.
  *
+ * @internal
```

Comment:
Hmm, I just had a thought. I think if we label something as internal it's skipped by the breaking change detector :-(

But i don't know another way we can say "don't use this class directly" that wouldn't affect the check.

Reply:
Hopefully it's smart enough to see the breaking changes in the non-internal surfaces.  I'll have to test it out

Reply:
I wonder if the Empty Client extending the BaseClient would give the BCD a non-internal surface to vet...

Reply:
I just checked, and the BC breaking client throws an error if it finds a non-internal class extending that parent class!

So my fears are invalid, and everything is great.

Reply:
Woo! thanks for raising and for checking :)